tests/bluetooth/controller/mock_ctrl/src/ull_periph.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Is `periph` the abbreviation we want to go with?

I suggest to write it out complete as `peripheral`.

Reply:
We'll consider this after this PR is merged into main

Reply:
Fixed in PR #40458